core/chains/evm/client/node.go

```diff
@@ -594,15 +722,25 @@ func (n *node) SuggestGasPrice(ctx context.Context) (price *big.Int, err error)
 	lggr := n.newRqLggr(switching(n))
 
 	lggr.Debug("RPC call: evmclient.Client#SuggestGasPrice")
+	start := time.Now()
 	if n.http != nil {
 		price, err = n.http.geth.SuggestGasPrice(ctx)
 		err = n.wrapHTTP(err)
 	} else {
 		price, err = n.ws.geth.SuggestGasPrice(ctx)
 		err = n.wrapWS(err)
 	}
-
-	n.logResult(lggr, err, "price", price)
+	duration := time.Since(start)
+
+	n.logResult(lggr, err, duration, n.getRPCDomain(), "SuggestGasPrice",
+		"price", price,
+		"duration", duration,
+		"rpcDomain", n.getRPCDomain(),
+		"name", n.name,
+		"chainID", n.chainID,
+		"sendOnly", false,
```

Comment:
These fields are unnecessary I think, they should be already put onto the node logger

Reply:
Oh good catch, they are indeed redundant.